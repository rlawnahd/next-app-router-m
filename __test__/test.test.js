function sum(a, b) {
    return a + b;
}
import Home from '@/app/(afterLogin)/home/page';

test('adds 1 + 2 to equal 3', () => {
    expect(sum(1, 2)).toBe(3);
});

import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';

describe('Home', () => {
    it('renders a heading', () => {
        render(<Home />);

        const heading = screen.getByRole('heading', { level: 1 });

        expect(heading).toBeInTheDocument();
    });
});
it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
});
