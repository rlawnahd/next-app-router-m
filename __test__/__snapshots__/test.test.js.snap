// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main>
    홈페이지
  </main>
</div>
`;
